lib/requests/reset-storage.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Please follow the naming convention from before, i.e. postfix the filename with `-request`.

Reply:
fixed (y)